components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/select-attribute.jsp

```diff
@@ -266,20 +267,20 @@
                 createInputs(paths[i].name);
             }
         }
-        document.attributeValueForm.action = "<%=returnPage%>.jsp?category="
-                + '<%=category%>' +"&ruleId=" + '<%=ruleId%>' ;
+        document.attributeValueForm.action = "<%=Encode.forUriComponent(returnPage)%>.jsp?category="
+                + '<%=Encode.forUriComponent(category)%>' +"&ruleId=" + '<%=Encode.forUriComponent(ruleId)%>' ;
         document.attributeValueForm.submit();
     }
 
     function doCancel(){
         preSubmit();
-        document.attributeValueForm.action = "<%=returnPage%>.jsp?ruleId=" + '<%=ruleId%>';
+        document.attributeValueForm.action = "<%=Encode.forUriComponent(returnPage)%>.jsp?ruleId=" + '<%=Encode.forUriComponent(ruleId)%>';
```

Comment:
returnPage is not included in a query string. Encode for javascript block instead